h/static/scripts/directive/test/annotation-share-dialog-test.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+var angular = require('angular');
```

Comment:
The format of imports as the top of JS files should be:

```js
'use strict';

<external imports>

<local imports>

<declarations>
```

With a blank line between each section.